test/elixir_analyzer/exercise_test/feature_test.exs

```diff
@@ -382,4 +382,227 @@ defmodule ElixirAnalyzer.ExerciseTest.FeatureTest do
       ]
     end
   end
+
+  describe "types of blocks" do
+    defmodule Blocks do
+      use ElixirAnalyzer.ExerciseTest
+
+      feature "single line" do
+        comment "single line"
+
+        form do
+          :a
+        end
+      end
+
+      feature "two lines" do
+        comment "two lines"
+
+        form do
+          :a
+          :b
+        end
+      end
+
+      feature "single line block" do
+        comment "single line block"
+
+        form do
+          !:a
+        end
+      end
+    end
+
+    defmodule BlocksTest do
+      use ElixirAnalyzer.ExerciseTestCase,
+        exercise_test_module: Blocks
+
+      test_exercise_analysis "empty",
+        comments: ["single line block", "two lines", "single line"] do
+        defmodule Module do
+        end
+      end
+
+      test_exercise_analysis "two lines",
+        comments: ["single line block"],
+        comments_exclude: ["single line", "two lines"] do
+        defmodule Module do
+          def foo do
+            :a
+            :b
+          end
+        end
+      end
+
+      test_exercise_analysis "single line block",
+        comments: ["two lines"],
+        comments_exclude: ["single line", "single line block"] do
+        defmodule Module do
+          def foo do
+            !:a
+          end
+        end
+      end
+    end
+  end
+
+  describe "other features" do
+    defmodule Coverage do
+      use ElixirAnalyzer.ExerciseTest
+
+      feature "skip" do
+        comment "skip"
+        status :skip
+
+        form do
+          :ok
+        end
+      end
+
+      feature "any" do
+        comment "any"
+        find :any
+
+        form do
+          :ok
+        end
+
+        form do
+          :error
+        end
+      end
+
+      feature "all" do
+        comment "all"
+        find :all
+
+        form do
+          :ok
+        end
+
+        form do
+          :error
+        end
+      end
+
+      feature "suppress_if" do
+        comment "suppress_if"
+        suppress_if "all", :pass
+
+        form do
+          :error
+        end
+      end
+
+      feature "depth 2" do
+        comment "depth 2"
+        depth 2
+
+        form do
+          :ok
+        end
+      end
+
+      feature "depth 3" do
+        comment "depth 3"
+        depth 3
+
+        form do
+          :ok
+        end
+      end
+
+      feature "meta" do
+        comment "meta"
+        meta(keep_meta(true))
```

Comment:
Same here, do we need this `meta` field? 
I can't imagine an application where we would want to keep the meta information. In fact I'm pretty sure that setting `keep_meta(true)` is equivalent to "never match this form".

Reply:
I think we can delete it. Forcing meta to match would mean forcing line numbers to match, which doesn't seem useful to me for anything. Adding an empty new line anywhere would mess up the whole analysis? Nah.

Reply:
Ok, I'll get rid of it. In a different PR :)